src/status_im/data_store/realm/schemas/account/v19/core.cljs

```diff
@@ -73,6 +73,12 @@
    ["phone"] {:content-command-ref ["console" :response 50 "phone"]}
    ["confirmation-code"] {:content-command-ref ["console" :response 50 "confirmation-code"]}})
 
+(def transactor-requests->new-props
+  {;; former transactor-personal request
+   ["send" 1] {:content-command-ref ["transactor" :response 83 "send"]}
+   ;; former transactor-group request
+   ["send" 2] {:content-command-ref ["transactor" :response 85 "send"]}})
```

Comment:
What's magical about 83 and 85?

Reply:
This is bitmask (as a decimal number) computed from command scope array on js side.
I'm not super thrilled about bitmasking as well (it's not like we are coding software for 8bit micro-controller and need to save every bit of memory), but as the javascript is unfortunately not supporting sets as a first class collection, we have to somehow encode (scopes) array in a way which produces constant result independent of the order of elements in it (or duplicates).

Reply:
Oh I see, thanks for the explanation. I realize this is out of scope of PR to improve this, but do you know if there is an issue to improve this? I'm certain we can come up with some better than decimal bitmasks to enode scopes.